docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+  frontend:
+    build: frontend
+    ports:
+      - "4200:80"
+    networks:
+      - front-tier
+      - back-tier
+  
+  backend:
+    build: backend
+    ports:
+      - "8080:8080"
+    networks:
+      - back-tier
+
+networks:
+  front-tier: {}
+  back-tier: {}
```

Comment:
Is there a reason to have two networks?

Reply:
Kurde, chyba nie. Jakoś tak poszło z tutorialem. Usunę to.